python/python/tests/tests_server_modules/search/test_ft_create.py

```diff
@@ -43,13 +43,15 @@ async def test_ft_create(self, glide_client: GlideClusterClient):
             glide_client, index, fields, FtCreateOptions(DataType.HASH, prefixes)
         )
         assert result == OK
+        assert await ft.dropindex(glide_client, indexName=index) == OK
```

Comment:
this almost feels like they should be separate tests, and we should apply dropindex after each test.

Reply:
Noted this to consider this when I cleanup the tests after all commands and move them to a common file.